iotp-adaptor/src/main/webapp/rootservices_rdfxml.jsp

```diff
@@ -40,8 +41,10 @@ String about = (String) request.getAttribute("about");
 	<oslc_am:amServiceProviders rdf:resource="<%= catalogUri %>" />
 	<oslc_rm:rmServiceProviders rdf:resource="<%= catalogUri %>" />
 	<oslc_cm:cmServiceProviders rdf:resource="<%= catalogUri %>" />
+	<oslc_cm:cmServiceProviders rdf:resource="<%= catalogUri %>" />
```

Comment:
Why do you have two cmServiceProviders with the same URI?

Reply:
It was a mistake, I think I did a double copy when adding the `oslc_config:cmServiceProviders` in the rootservices jsp.